src/serialization/serialization.rs

```diff
@@ -176,13 +183,19 @@ mod benchmarks;
 mod v2_serializer;
 pub use self::v2_serializer::{V2Serializer, V2SerializeError};
 
+#[path = "v2_deflate_serializer.rs"]
```

Comment:
Hmm.. Reading issue https://github.com/rust-lang/rust/issues/37872, it seems like this is caused by us using `#[path]` elsewhere, which sort "taints" the path such that everything needs `#[path]` annotations. Weird... And unfortunate..